app/index.html

```diff
@@ -71,6 +71,36 @@ <h1 class="center-text primary-text-color primary-font heading u-margin">Hey the
     <section id="works" class="u-margin">
         <h2 class="primary-text-color primary-font heading">My Recent works</h2>
         <div class="indicator"></div>
+
+        <!--- modal start --->
+        <div id="modal-wrapper">
+            <div id="modal" class="primary-font primary-text-color">
+                <div id="modal-snapshot">
+                    <div id="modal-cancel">
+                        <img src="/app/assets/icn-cancel-white.png" alt="cancel button">
```

Comment:
- [ ] Kindly try to fix this image cancel icon link, actually, the icon is not showing up. check here :point what I can see

![image](https://user-images.githubusercontent.com/65068771/151159509-b1444ad0-c29b-4d03-b3f1-4295f6ed9d30.png)